Source/SwiftLintFramework/Rules/Lint/UnusedSetterArgumentRule.swift

```diff
@@ -0,0 +1,234 @@
+import Foundation
+import SourceKittenFramework
+
+public struct UnusedSetterArgumentRule: ConfigurationProviderRule, AutomaticTestableRule {
+    public var configuration = SeverityConfiguration(.warning)
+
+    public init() {}
+
+    public static let description = RuleDescription(
+        identifier: "unused_setter_argument",
+        name: "Unused Setter Argument",
+        description: "Setter argument is not used.",
+        kind: .lint,
+        nonTriggeringExamples: [
+            """
+            var aValue: String {
+                get {
+                    return Persister.shared.aValue
+                }
+                set {
+                    Perister.shared.aValue = newValue
```

Comment:
Nit: `Perister` here and a few other places

Reply:
Copied from #1136 😅